### PR TITLE
Adds the Unathi Kataphracts as a response team

### PR DESCRIPTION
**Pull request by Geevies**
The kataphracts can be played by anyone, even unwhitelisted players (hopefully). They're a more melee-focused team, if you spot them, gun them down at a range.

This PR also adds the functionality to implement response teams of a non-human type without the need for the person joining it to have the whitelist.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/datums/outfits/ert/kataphract_ert.dm [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+		var/obj/item/clothing/shoes/magboots/boots = new(H)
+		H.equip_to_slot_if_possible(boots, slot_shoes)
+
+/datum/outfit/admin/ert/kataphract/leader
+	name = "Kataphract Knight"
+
+	glasses = /obj/item/clothing/glasses/thermal
+
+/datum/outfit/admin/ert/kataphract/leader/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	if(H && H.wear_suit)
```
Comment (skull132) [inline code/datums/outfits/ert/kataphract_ert.dm]:
```suggestion
	if(H?.wear_suit)
```

File: code/datums/outfits/ert/kataphract_ert.dm [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+		var/obj/item/clothing/accessory/poncho/red/big/poncho = new(H)
+		var/obj/item/clothing/suit/space/void/kataphract/S = H.wear_suit
+		S.attach_accessory(null, poncho)
+	if(H && H.shoes)
```
Comment (skull132) [inline code/datums/outfits/ert/kataphract_ert.dm]:
```suggestion
	if(H?.shoes)
```

File: code/datums/outfits/ert/kataphract_ert.dm (hunk 1 above)
Comment (Geevies) [reply code/datums/outfits/ert/kataphract_ert.dm]: what kinda arcane bullshit

File: code/modules/ghostroles/spawner/human/ert.dm [hunk 3]
```diff
@@ -154,4 +154,32 @@
...
+	desc = "A Zo'saa (squire) trained in medicine from the local Kataphract guild."
+	max_count = 1
+	outfit = /datum/outfit/admin/ert/kataphract/specialist
+	mob_name_prefix = "Zosaa "
+
+/datum/ghostspawner/human/ert/kataphract/leader
+	name = "Kataphract Knight"
+	short_name = "katlead"
+	desc = "A brave Saa (Knight) of the local Kataphract Guild. Two together operate as leaders of the team."
+	max_count = 2
```
Comment (NonQueueingMatt) [inline code/modules/ghostroles/spawner/human/ert.dm]: Two leaders?

Comment (Geevies) [reply code/modules/ghostroles/spawner/human/ert.dm]: Yes.

**Comment by Kaedwuff** [comment]
What are we going to do if people play unathi out of character or like meme lizards?

**Comment by Geevies** [comment]
> 
> 
> What are we going to do if people play unathi out of character or like meme lizards?

Administrative action, probably.

**Pull request merged**